Gordon360/Services/NewsService.cs

```diff
@@ -292,9 +313,9 @@ private string GetImageURL(string filename)
         /// <returns>true if unapproved, otherwise throws some kind of meaningful exception</returns>
         private static bool VerifyUnapproved(StudentNews newsItem)
         {
-            // Note: This check has been duplicated from StateYourBusiness because we do not SuperAdmins
-            //    to be able to delete expired news, this should be fixed eventually by removing some of
-            //    the SuperAdmin permissions that are not explicitly given
+            // Note: These checks have been duplicated from StateYourBusiness because we do not want
+            //    SuperAdmins to be able to delete expired news, this should be fixed eventually by
+            //    removing some of the SuperAdmin permissions that are not explicitly given
             if (newsItem.Accepted == null)
             {
                 throw new ResourceNotFoundException() { ExceptionMessage = "The news item acceptance status could not be verified." };
```

Comment:
```suggestion
                throw new ResourceNotFoundException() { ExceptionMessage = "The news item's accepted status could not be verified." };
```
Could we update this error to be consistent with your messaging? I prefer the way that you wrote it to the way it is here.